### PR TITLE
Rvv fix 2020 02 14

### PR DESCRIPTION
**Pull request by chihminchao**
fix some vector instruction corner cases

### CODE REVIEW

File: riscv/insns/vmsgtu_vi.h [hunk 1]
```diff
@@ -1,5 +1,6 @@
 // vsgtu.vi  vd, vd2, simm5
 VI_VI_ULOOP_CMP
 ({
-  res = vs2 > simm5;
+  uint64_t imm = (int8_t)(simm5 << 3) >> 3;
```
Comment (aswaterman) [inline riscv/insns/vmsgtu_vi.h]: The `v_simm5()` function already does this sign-extension to uint64_t; why do you need to do it again?

File: riscv/insns/vmsgtu_vi.h [hunk 2]
```diff
@@ -1,5 +1,6 @@
 // vsgtu.vi  vd, vd2, simm5
 VI_VI_ULOOP_CMP
 ({
-  res = vs2 > simm5;
+  uint64_t imm = (int8_t)(zimm5 << 3) >> 3;
```
Comment (aswaterman) [inline riscv/insns/vmsgtu_vi.h]: Just call v_simm5() here instead?

**Review by aswaterman — APPROVED** [review]
See my one comment, otherwise looks good, thanks!

File: riscv/insns/vmsgtu_vi.h (hunk 2 above)
Comment (chihminchao) [reply riscv/insns/vmsgtu_vi.h]: good suggestions.

**Comment by aswaterman** [comment]
Thank you, @chihminchao!

**Pull request merged**